services/cacheService.js

```diff
@@ -61,4 +62,4 @@ const getService = () => {
   return defaultService;
 };
 
-export default getService;
+module.exports = getService;
```

Comment:
In line with my comments above, have this function return the `defaultService` if redis isn't configured, return the results of `getRedisService` if it is.